<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Isometric Projections</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>1081</width>
     <height>751</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="slider_x">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>359</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>30</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider_y">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>359</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>30</number>
   </property>
  </widget>
  <widget class="QSlider" name="slider_z">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>50</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>359</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 20pt &quot;Ubuntu&quot;;
color: rgb(204, 0, 0)</string>
   </property>
   <property name="text">
    <string>X angle</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 20pt &quot;Ubuntu&quot;;
color: rgb(204, 0, 0)</string>
   </property>
   <property name="text">
    <string>Y angle</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 20pt &quot;Ubuntu&quot;;
color: rgb(204, 0, 0)</string>
   </property>
   <property name="text">
    <string>Z angle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_reset">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
